src/services/doppler-legacy-client.ts

```diff
@@ -353,7 +353,7 @@ export interface AdvancedPayOptions {
 export interface PlanModel {
   id: number;
   description: string;
-  emailsByMonth?: number;
+  emailsQty?: number;
```

Comment:
Entinedo que este cambio es para soportar tanto el prepago como el mensual. Creo que son dos conceptos diferentes no me parece buena idea unificarlos en un solo item.

Reply:
Yes here we can have `credits` property mapped apart.

Reply:
In that case I prefer the old name, Am I missing something?

Reply:
Credits are forever and not neccesarily emails. Emails are by month and you can exceed at a cost, subscribers is a limit for the plan.